resources/views/cart/sidebar.blade.php

```diff
@@ -11,20 +11,24 @@
         <dt>@{{ name }}</dt>
         <dd>@{{ value | price }}</dd>
     </div>
-    <div v-if="cart.shipping_addresses.length && cart.shipping_addresses[0].selected_shipping_method">
```

Comment:
`cart.shipping_addresses[0].selected_shipping_method` should stay the primary value I think; `cart.shipping_method` as fallback?